docs/labwc-actions.5.scd

```diff
@@ -133,6 +133,9 @@ Actions are used in menus and keyboard/mouse bindings.
 	when defining window-rules for desktop-management tools that do not
 	support the wlr-layer-shell protocol.
 
+*<action name="ToggleOmnipresent" />*
+	Toggle omnipresent for the focused window.
```

Comment:
Suggest something like
> Toggle visible-on-all-workspaces for the focused window.

I think we should not assume that the word "Omnipresent" is known widely to mean on-all-workspaces. My documentation skills aren't that great though so a better text is also welcome :)